.appveyor.yml

```diff
@@ -5,13 +5,12 @@ environment:
 
   # Workaround for https://github.com/conda/conda-build/issues/636
   PYTHONIOENCODING: "UTF-8"
+  CONDA_INSTALL_LOCN: "C:\\Miniconda36-x64"
```

Comment:
better to leave this as Miniconda3-x64 - that's a symlink to the current miniconda3 install on appveyor.  It'll be better to keep up-to-date that way (fewer updates, potentially)